aws-alb-ingress-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,130 @@
+{{ $root := . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "aws-alb-ingress-controller.fullname" $root }}
+  labels: {{ include "aws-alb-ingress-controller.labels" $root | nindent 4 }}
+    {{- range $key, $value :=.Values.deployment.labels }}
+    {{ $key }}: {{ tpl $value $root | quote }}
+    {{- end }}
+  {{- range $key, $value := .Values.deployment.annotations }}
+  annotations:
```

Comment:
@cw-sakamoto 
I think it's better to place it outside of `range`.

Reply:
https://github.com/chatwork/charts/pull/147